include/uzfs_rebuilding.h

```diff
@@ -22,13 +22,34 @@
 #ifndef	_UZFS_REBUILDING_H
 #define	_UZFS_REBUILDING_H
 
+#include <sys/queue.h>
+#include <netinet/in.h>
+#include <arpa/inet.h>
+
 #ifdef __cplusplus
 extern "C" {
 #endif
 #define	IO_DIFF_SNAPNAME		".io_snap"
 #define	REBUILD_SNAPSHOT_SNAPNAME	"rebuild_snap"
 #define	REBUILD_SNAPSHOT_CLONENAME	"rebuild_clone"
 
+/*
+ * rebuild statistics
+ */
+typedef struct rebuild_stats {
+	TAILQ_ENTRY(rebuild_stats) stat_next;
```

Comment:
Is it because we may rebuild a replica from more than one replica at a time ?

Reply:
yes